requirements/base.in

```diff
@@ -8,3 +8,5 @@ djangorestframework
 edx-celeryutils
 edx-django-utils>=3.12.0   # Utilities for caching and monitoring
 simplejson
+setuptools
```

Comment:
We should try adding the `setuptools` and `wheel` in the tox dependencies instead if it works.

Reply:
@awais786 we should do this, othrewise, setuptools and wheel will become dependencies of this project for use which is not what we want.

Reply:
Looks like you've already added it to the tox file so maybe it just needs to be removed from here?